ui/src/playlist/PlaylistSongs.js

```diff
@@ -162,6 +163,7 @@ const PlaylistSongs = ({ playlistId, readOnly, ...props }) => {
               {isDesktop && <AlbumLinkField source="album" />}
               {isDesktop && <TextField source="artist" />}
               <DurationField source="duration" className={classes.draggable} />
+              <QualityInfo label="Quality" />
```

Comment:
Labels need to be translated.

Reply:
Translated? Did not understood that

Reply:
You need to add the "Quality" label to the `en.json` file, and set a `source="quality"` attribute here. Like the other fields